en/docs/getting-started/basic-architecture.md

```diff
@@ -1,73 +1,72 @@
-#Basic Architecture and Key Components
+# Basic Architecture and Key Components
 
-![Basic Architecture]({{base_path}}/assets/img/get_started/basic-architecture.png)
+[![Basic Architecture]({{base_path}}/assets/img/get_started/basic-architecture.png)]({{base_path}}/assets/img/get_started/basic-architecture.png)
 
 ## Management Plane
 
-The management plane is where API creation and API management takes place.  It consists of portals (publisher, developer portal, admin portal) for users to create and manage APIs, implement rate-limiting policies, monitor and monetization, etc. It also provides a set of APIs to interact with external tools like API Controller. The management plane includes API Analytics dashboards, displaying various business insights. . 
+The Management Plane is where API creation and API management takes place.  It consists of portals (Publisher, Developer Portal, Admin Portal) for users to create and manage APIs, implement rate limiting policies, monitor, and monetization, etc. It also provides a set of APIs to interact with external tools like API Controller. The Management Plane includes API Analytics dashboards, displaying various business insights.
 
 ### Components
 
 #### API Publisher
 
-![API Publisher]({{base_path}}/assets/img/learn/overviewpage-rest-api.jpg)
+[![API Publisher]({{base_path}}/assets/img/learn/overviewpage-rest-api.jpg)]({{base_path}}/assets/img/learn/overviewpage-rest-api.jpg)
```

Comment:
This screenshot is outdated. The menu named "Scopes" should be "Local Scopes"

<img width="1032" alt="Screenshot 2020-12-09 at 17 29 04" src="https://user-images.githubusercontent.com/5195851/101627502-73356b80-3a44-11eb-9e30-c1db5cd2aa7e.png">